helm/alfresco-process-infrastructure/README.md

```diff
@@ -35,6 +35,10 @@ Source code can be found [here](https://github.com/Alfresco/alfresco-process-inf
 | alfresco-admin-app.ingress.path | string | `"/admin"` |  |
 | alfresco-admin-app.nameOverride | string | `"alfresco-admin-app"` |  |
 | alfresco-deployment-service.alfresco-content-services.enabled | bool | `true` |  |
+| alfresco-deployment-service.applications.connectors.emailConnector.host | string | `""` |  |
```

Comment:
@SohelSaiyed in my mind i wanted to add the info about email connector properties in the description section here, what do you think?

Reply:
not sure the description can be added here, as we wanted to keep the value as empty and this doc is auto-generated by helm-docs.

Reply:
oh ok, and how is it possible to populate the description column then?
Maybe we can fit it in there